runtime/runtime.rs

```diff
@@ -97,6 +97,13 @@ pub async fn run_js_module(
     module_specifier: &ModuleSpecifier,
     bootstrap_options: &BootstrapOptions,
 ) -> Result<(), AnyError> {
+    if !Regex::new(r"^[0-9a-fA-F]{88}$")
+        .unwrap()
```

Comment:
Let's move the Regex initialisation/compilation outside of the `run_js_module` function. While this is not a hot code path, I'd like to avoid preliminary pessimization.

See https://docs.rs/regex/latest/regex/#avoid-re-compiling-regexes-especially-in-a-loop

```rs
use {
    once_cell::sync::Lazy,
    regex::Regex,
};

fn some_helper_function(haystack: &str) -> bool {
    static RE: Lazy<Regex> = Lazy::new(|| Regex::new(r"...").unwrap());
    RE.is_match(haystack)
}
```